### PR TITLE
Bugfix on alerts generation for yamls

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

Some alerts were missing and some had wrong `os` tags. Should be fixed.

### CODE REVIEW

**Review by ilyam8 — APPROVED** [review]

File: collectors/ebpf.plugin/multi_metadata.yaml [hunk 1]
```diff
@@ -705,7 +705,14 @@ modules:
     troubleshooting:
       problems:
         list: []
-    alerts: []
+    alerts:
+      - name: sync_freq
+        link:
```
Comment (Dim-P) [inline collectors/ebpf.plugin/multi_metadata.yaml:710]: @Ancairon  Can you please check this error?

File: collectors/proc.plugin/multi_metadata.yaml [hunk 2]
```diff
@@ -4632,7 +4734,12 @@ modules:
     troubleshooting:
       problems:
         list: []
-    alerts: []
+    alerts:
+      - name: linux_power_supply_capacity
+        link: 
+          https://github.com/netdata/netdata/blob/master/health/health.d/linux_power_supply.conf
```
Comment (thiagoftsm) [inline collectors/proc.plugin/multi_metadata.yaml:4740]: @Ancairon we have another alert here

File: collectors/ebpf.plugin/multi_metadata.yaml (hunk 1 above)
Comment (Ancairon) [reply collectors/ebpf.plugin/multi_metadata.yaml:710]: Some weird yamllint error, will try and see why it adds a newline there..

File: collectors/proc.plugin/multi_metadata.yaml (hunk 2 above)
Comment (Ancairon) [reply collectors/proc.plugin/multi_metadata.yaml:4740]: these warnings are not entirely true, the yaml validates fine, will have to see if I can prevent this through ruamel.yaml.

**Comment by Ancairon** [comment]
yamllint complains either about trailing spaces or about long lines, probably will fix the warnings manually on the previous commit.

**Comment by Ancairon** [comment]
Checks now pass, but I don't like how multiline strings are just split, for example in beanstalk.. I guess we will have to live with this and just fix it while filling in the yamls

**Review by thiagoftsm — APPROVED** [review]

**Review by Dim-P — APPROVED** [review]

**Pull request merged**